src/js/stores/MarathonStore.js

```diff
@@ -159,6 +159,30 @@ var MarathonStore = Store.createStore({
     return this.get('apps')[name];
   },
 
+  getServiceNameFromTaskID: function (taskID) {
```

Comment:
As this method is only parsing the task id I would rather move it to the `Task` struct or an util, as suggested by @MatApple.